obelisk/cpp/zoo/robot/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
this is a pretty low minimum version. let's stick to 3.16 - we're working to standardize this across our codebase.